tensorflow_addons/activations/rrelu_test.py

```diff
@@ -52,7 +52,15 @@ def test_theoretical_gradients(self, dtype):
         for training in [True, False]:
             with self.subTest(training=training):
                 theoretical, numerical = tf.test.compute_gradient(
-                    lambda x: rrelu(x, lower, upper, training=training), [x])
+                    lambda x: rrelu(
+                        x, lower, upper, training=training, seed=111111), [x])
+                if training is True and tf.test.is_gpu_available() is False:
```

Comment:
There is a bug in compute_gradient when it runs on CPU and GPU

Reply:
I am fine with this :-) Maybe use
```
if training and not tf.test.is_gpu_available:
```
and add a TODO like
```
# TODO: investigate the difference between CPU and GPU
```